packages/react-menu/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@fluentui/react-menu",
+  "version": "0.1.0",
+  "description": "Fluent UI menu component",
+  "private": true,
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.1",
+    "@fluentui/scripts": "^1.0.0",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/jest": "~24.9.0",
+    "@types/react": "16.9.42",
+    "@types/react-dom": "16.9.10",
+    "@types/react-test-renderer": "^16.0.0",
+    "@types/webpack-env": "1.16.0",
+    "enzyme": "~3.10.0",
+    "enzyme-adapter-react-16": "^1.15.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0"
+  },
+  "dependencies": {
+    "@fluentui/keyboard-key": "^0.2.13",
+    "@fluentui/react-compose": "^1.0.0-beta.11",
+    "@fluentui/react-make-styles": "^0.2.1",
+    "@fluentui/react-theme-provider": "^1.0.0-beta.19",
```

Comment:
Should this not be a dev dep since it's used only in the StoryBook?

Reply:
On second thought, since the StoryBook is not part of this package, maybe we don't need this at all?

Reply:
Up to those of you working more closely on convergence whether these deps should stay or not, but if you decide they should be removed, please update the template too (probably in a separate PR)